proto/stride/records/records.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+package stride.records;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v9/x/records/types";
+
+message UserRedemptionRecord {
+  string id = 1; // {chain_id}.{epoch}.{sender}
+  string sender = 2;
+  string receiver = 3;
+  string amount = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string denom = 5;
+  string host_zone_id = 6;
+  uint64 epoch_number = 7;
+  bool claim_is_pending = 8;
+}
+
+message DepositRecord {
+  uint64 id = 1;
+  string amount = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string denom = 3;
+  string host_zone_id = 4;
+  enum Status {
+    // in transfer queue to be sent to the delegation ICA
+    TRANSFER_QUEUE = 0;
+    // transfer in progress (IBC packet sent, ack not received)
+    TRANSFER_IN_PROGRESS = 2;
+    // in staking queue on delegation ICA
+    DELEGATION_QUEUE = 1;
+    // staking in progress (ICA packet sent, ack not received)
+    DELEGATION_IN_PROGRESS = 3;
+  }
+  enum Source {
+    STRIDE = 0;
+    WITHDRAWAL_ICA = 1;
+  }
+  Status status = 6;
+  uint64 deposit_epoch_number = 7;
+  Source source = 8;
+
+  reserved 5;
+}
+
+message HostZoneUnbonding {
+  enum Status {
+    // tokens bonded on delegate account
+    UNBONDING_QUEUE = 0;
+    UNBONDING_IN_PROGRESS = 3;
+    // unbonding completed on delegate account
+    EXIT_TRANSFER_QUEUE = 1;
+    EXIT_TRANSFER_IN_PROGRESS = 4;
+    // transfer success
+    CLAIMABLE = 2;
+  }
+  string st_token_amount = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string native_token_amount = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string denom = 3;
+  string host_zone_id = 4;
+  uint64 unbonding_time = 5;
+  Status status = 6;
+  repeated string user_redemption_records = 7;
+}
+
+message EpochUnbondingRecord {
+  uint64 epoch_number = 1;
+  repeated HostZoneUnbonding host_zone_unbondings = 3;
+  reserved 2;
+}
+
+message LSMTokenDeposit {
+  enum Status {
+    DEPOSIT_PENDING = 0;
+    TRANSFER_IN_PROGRESS = 1;
```

Comment:
Checked that in the generated pb file doing the statuses this way may change the term we refer to in go to LSMTokenDeposit_TRANSFER_IN_PROGRESS but the actual string itself does not change and will still be "TRANSFER_IN_PROGRESS".  This is important for the CLI and http GET interfaces when filtering by status since that is what the user would have to type in.  No issue, should be good, just calling it out.